@kiva/kv-components/vue/KvIntroductionLoanCard.vue

```diff
@@ -228,6 +228,7 @@
 					:loan="loan"
 					variation="matched-loan"
 					class="tw-text-center !tw-text-brand"
+					:kv-track-function="kvTrackFunction"
```

Comment:
Why was this added?

Reply:
Please remove the prop from `KvLoanTag`, since you're right, it's no longer being used.